app/assets/stylesheets/components.scss

```diff
@@ -903,6 +909,12 @@ a.status__content__spoiler-link {
   .fa {
     color: $color4;
   }
+
+  .notification__avatar {
+    display: inline-block;
+    vertical-align: middle;
+    margin-right: 5px;
+  }
```

Comment:
This is pretty nitpicky, but seeing as both `.status__avatar-boost` and `.notification__avatar` have the exact same styles, do you think you could combine them?